app/assets/v2/css/users.css

```diff
@@ -22,6 +22,10 @@
   }
 }
 
+.users .dropdown-menu {
```

Comment:
this need to be removed, will affect the nav dropdowns and every dropdown on this page, that is a bootstrap class and that component is used inside others components.

Reply:
updated